File: customview/src/main/java/com/example/customview/widget/banner/listener/OnPageClickListener.java
package com.example.customview.widget.banner.listener;

import android.view.View;
/**
 * 用于约束 On Page Click 回调能力的接口。
 */

public interface OnPageClickListener {

    /**
     * item 点击事件
     *
     * @param view     view
     * @param position position
     */
    void onPageClick(View view, int position);

}

File: customview/src/main/java/com/example/customview/widget/banner/listener/OnPageClickListener.kt
package com.example.customview.widget.banner.listener

import android.view.View

/**
 * 用于约束 On Page Click 回调能力的接口。
 */
fun interface OnPageClickListener {
    /**
     * item 点击事件
     *
     * @param view     view
     * @param position position
     */
    fun onPageClick(view: View, position: Int)
}
